api/conf/conf.yaml

```diff
@@ -17,10 +17,15 @@
 
 conf:
   listen:
-    host: 127.0.0.1     # `manager api` listening ip or host name
     port: 9000          # `manager api` listening port
   allow_list:           # If we don't set any IP list, then any IP access is allowed by default.
     - 127.0.0.0/24
+    - 10.0.0.0/8
```

Comment:
only one `127.0.0.0/24` is enough.

we can add more comment to show how to set it

Reply:
updated.